data/templates/nginx/server.tpl.conf

```diff
@@ -20,41 +20,39 @@ server {
 }
 
 server {
-    # Disabling http2 for now as it's causing weird issues with curl
-    #listen 443 ssl http2;
-    #listen [::]:443 ssl http2;
-    listen 443 ssl;
-    listen [::]:443 ssl;
+    # curl is fixed
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
```

Comment:
Ok, mais a besoin d'être abordé au cours d'une réunio pour évaluer les risques de compatibilité.

Reply:
Ok faite le test on la fait

Reply:
quand je parle de compatibilité ce ne sont pas que les navigateurs mais les applications.

Reply:
je parle de l'app cela à était fixé en juin 2018 sur le github

Reply:
Je pense que la meilleur solution pour valider cela est de faire une testing avec cette option et de tester un max d'app avec la CI.

I think that the best solution to validate this option is to make a testing with this option and to test a lot of apps with the CI and to see if there are any regression.

Reply:
Oui j'ai plusieurs apps qui on aucun soucis ^^ emby / plex / pihole / onlyoffice et bien d'autres

Reply:
The problem is generaly on the install or upgrade.

Reply:
d'ailleurs j'ai pas encore trouvé d'apps qui ne fonctionne pas avec c'est un point qui est tellement standard aujourd’hui

Reply:
Not really sur, curl failled without the PR https://github.com/YunoHost/yunohost/pull/547, while we tried to install wordpress

Reply:
im go tro try to install wp

Reply:
Im check APPS LIST ###  

1. Nextcloud OK

Reply:
But a simple sudo yunohost app install my app work